Makefile

```diff
@@ -70,17 +82,27 @@ clean: ## Clean project: remove created binaries and apps
 	-rm -rf ./apps
 	-rm -f ./skywire-visor ./skywire-cli ./setup-node
 
+clean-windows: ## Clean project: remove created binaries and apps
+	powershell -Command Remove-Item -Path ./apps -Force -Recurse
+	powershell -Command Remove-Item -Path .\skywire-visor.exe,.\skywire-cli.exe,.\setup-node.exe -Force
+
 install: ## Install `skywire-visor`, `skywire-cli`, `setup-node`
 	${OPTS} go install ${BUILD_OPTS} ./cmd/skywire-visor ./cmd/skywire-cli ./cmd/setup-node
 
+install-windows: ## Install `skywire-visor`, `skywire-cli`, `setup-node`
+	powershell 'Get-ChildItem .\cmd | % { ${OPTS} go install ${BUILD_OPTS} ./ $$_.FullName }'
+
 install-static: ## Install `skywire-visor`, `skywire-cli`, `setup-node`
 	${STATIC_OPTS} go install -trimpath --ldflags '-linkmode external -extldflags "-static" -buildid=' ./cmd/skywire-visor ./cmd/skywire-cli ./cmd/setup-node
 
 lint: ## Run linters. Use make install-linters first
 	${OPTS} golangci-lint run -c .golangci.yml ./...
 	# The govet version in golangci-lint is out of date and has spurious warnings, run it separately
 
-lint-windows-appveyor:
+lint-windows: ## Run linters. Use make install-linters-windows first
+	powershell 'golangci-lint run -c .golangci.yml ./...'
+
+lint-appveyor-windows: ## Run linters for appveyor only on windows
```

Comment:
This is giving an error
```
PS C:\Users\Erson\skywire> make lint-appveyor-windows
C:\Users\appveyor\go\bin\golangci-lint run -c .golangci.yml ./...
process_begin: CreateProcess(NULL, C:\Users\appveyor\go\bin\golangci-lint run -c .golangci.yml ./..., ...) failed.
make (e=2): The system cannot find the file specified.
make: *** [Makefile:106: lint-appveyor-windows] Error 2
```

Reply:
Have you tried to add `golangci-lint` to your own `%PATH%`, because that one is for appveyor

Reply:
and you should've run `make check-windows`

Reply:
Okay. Was just testing all the commands. Everything else works then.